Chess/game.h

```diff
@@ -25,11 +25,18 @@ public slots:
     void stopAction();
     void saveAction();
     void loadAction();
+    void tmpDraw();
+
+private:
+    void drawCurState();
+    void drawCommand();
+    void interruptCommand();
 
 private:
-    QObject *viewer_;       //for access to gui elements
-    //QObject *desk_;         //ptr to desk instance (game have 1 instance of desk during all life period)
-    Desk *desk_;         //ptr to desk instance (game have 1 instance of desk during all life period)
+    QObject *root_;     //for access to gui elements
+    Desk *desk_;        //ptr to desk instance (game have 1 instance of desk during all life period)
+    Command *command_;  //current command

```

Comment:
Raw pointers are evel. If something is one per aplication just incapsulate it in application instance.